server/views/pages/search/caSearch/caSearch.njk

```diff
@@ -165,7 +165,7 @@
     {
         attributes: {
             id: 'release-date-' + loop.index,
-            "data-sort-value": offender.releaseDate | dateToUnix
+            "data-sort-value": offender.releaseDate | formatCvlDateAsEurDate | dateToUnix
```

Comment:
These two filters together `formatCvlDateAsEurDate | dateToUnix` format the `releaseDate` then convert it to a unix value to enable accurate sorting by release date on the CA and PP search screens.